src/components/MishnaView/ManuscriptPopup.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { iManuscript, iManuscriptPopup } from '../../types/types';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import { setManuscriptsForChapter, setSublineData } from '../../store/actions/relatedActions';
+import ZoomImage from '../manuscripts/ZoomImage';
+import { getImageUrl } from '../../inc/manuscriptUtils';
+
+const sx = {
+  root: {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    bgcolor: 'background.paper',
+    borderRadius: '4px',
+    p: 1,
+  },
+};
+
+const mapStateToProps = (state) => ({
+  sublineData: state.related.sublineData,
+  currentRoute: state.navigation.currentRoute,
+  manuscriptsForChapter: state.related.manuscriptsForChapter,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  closeManuscriptPopup: () => {
+    dispatch(setSublineData(null));
+  },
+  setManuscriptsForChapter: (chapter: string, tractate: string) => {
+    dispatch(setManuscriptsForChapter(chapter, tractate));
+  },
+});
+
+export interface iProps {
+  sublineData: iManuscriptPopup;
+  currentRoute: any;
+  closeManuscriptPopup: () => void;
+  setManuscriptsForChapter: (chapter: string, tractate: string) => void;
+  manuscriptsForChapter: iManuscript[];
+}
+
+const ManuscriptPopup = (props: iProps) => {
+  const { sublineData, closeManuscriptPopup, currentRoute, setManuscriptsForChapter, manuscriptsForChapter } = props;
+  const [imageURL, setImageURL] = useState('');
+
+  useEffect(() => {
+    setImageURL(getImageUrl(manuscriptsForChapter, sublineData));
+  }, [sublineData?.subline.index, sublineData?.line]);
+
+  useEffect(() => {
+    setManuscriptsForChapter(currentRoute?.tractate, currentRoute?.chapter);
+  }, [currentRoute?.chapter, currentRoute?.tractate]);
+
+  return (
+    <Modal
```

Comment:
Would be better to use a <Dialog> component and set it to `fullwidth`
With a dialog we can also divide the modal into content and actions (later we will add buttons)

Reply:
Sorry, I didn't type one important word :)
I meant a <Dialog> component

https://mui.com/material-ui/react-dialog/#responsive-full-screen